packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -876,7 +886,9 @@ const DataTable = ({
         hasFilters ||
         hasChangeableDensity ||
         hasColumnVisibility ||
-        bulkActionMenuItems) && (
+        bulkActionMenuItems ||
+        additionalActionButton ||
+        additionalActionMenuItems) && (
```

Comment:
@jordankoschei-okta [this conditional logic](https://github.com/okta/odyssey/pull/2319/files#diff-6941e23192090c164e50b43565b68ad0df07073844d27ce5b9b5c802dd72eea9L876-R891) is getting pretty out of hand. Is there a way we can simplify this?